File: app/src/main/java/com/github/karthyks/socialgameworks/main/MainActivity.java
package com.github.karthyks.socialgameworks.main;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

import com.github.karthyks.socialgameworks.AppSession;
import com.github.karthyks.socialgameworks.R;


public class MainActivity extends AppCompatActivity {

  private TextView tvUserId;

  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    tvUserId = (TextView) findViewById(R.id.tv_user_id);
    tvUserId.setText(AppSession.getInstance(this).getAuthenticatedUser().getSteamId());
  }
}

File: app/src/main/java/com/github/karthyks/socialgameworks/main/MainActivity.kt
package com.github.karthyks.socialgameworks.main

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.TextView
import com.github.karthyks.socialgameworks.AppSession
import com.github.karthyks.socialgameworks.R


class MainActivity : AppCompatActivity() {

    private var tvUserId: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        tvUserId = findViewById(R.id.tv_user_id)
        tvUserId!!.text = AppSession.getInstance(this).authenticatedUser!!.steamId
    }


}
